standard_risk_tests.ipynb

```diff
@@ -0,0 +1,1116 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "model = {}\n",
+    "response = session.get(moc_url + 'model-manage/api/storedModels/' + model_selector.value)\n",
+    "if response.ok:\n",
+    "\tmodel = response.json()\n",
+    "\tdisplay(VBox([\n",
+    "\t\tHBox([Label(value=\"Model Name:\"), Label(model.get(\"modelMetaData\", {}).get(\"name\", \"Name not set\"))]),\n",
+    "\t\tHBox([Label(value=\"Description:\"), Label(model.get(\"modelMetaData\", \"\").get(\"description\", \"\"))]),\n",
+    "\t\tHBox([Label(value=\"Created By:\"), Label(model.get(\"createdBy\", \"Unknown\"))]),\n",
+    "\t\tHBox([Label(value=\"Created Date:\"), Label(model.get(\"createdDate\", \"Unknown\"))])\n",
+    "\t]))\n",
+    "\tdisplay(HTML(\"Model loaded and can be viewed <a href=\" + moc_url + \"#/models/model/\" + model[\"id\"] + \" target=\\\"_blank\\\" >here</a>.\"))\n",
+    "else:\n",
+    "\tprint(\"ERROR - The model could not be found, please update the example_model_name variable with the correct model name to examine\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## Check for valid schema\n",
+    "\n",
+    "Let's now check that the target model has the schema populated.  All target models for dashboards and standardized tests\n",
+    "will typically require an input schema definition.  This allows us to know what columns are used for labels, inputs, and\n",
+    "which columns are a protected class, as well as other pertinent information about the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "column_labels = []\n",
+    "input_schema = model.get(\"modelMetaData\", {}).get(\"inputSchema\", [])\n",
+    "if len(input_schema) != 1:\n",
+    "\tdisplay(HTML(\"The dashboard model requires exactly one input schema to be specified. Please go to the model's schema page in MOC, and generate and edit a valid schema for the input data\"))\n",
+    "else:\n",
+    "\tinput_schema = model.get(\"modelMetaData\", {}).get(\"inputSchema\", [])\n",
+    "\tresponse = session.post(moc_url + \"model-manage/api/schema/validate/?extended=true\", json = input_schema[0].get(\"schemaDefinition\", {}))\n",
+    "\tif response.ok:\n",
+    "\t\tdisplay(HTML(\"The example model contains a valid schema\"))\n",
+    "\t\tfor col in input_schema[0].get(\"schemaDefinition\", {}).get(\"fields\", []):\n",
+    "\t\t\tcolumn_labels.append(col.get(\"name\", \"\"))\n",
+    "\t\tdisplay(column_labels)\n",
+    "\telse:\n",
+    "\t\tdisplay(HTML(\"The dashboard model requires a valid extended schema definition. Please go to the model's schema page in MOC, and generate and edit a valid schema for the input data\"))\n",
```

Comment:
Please update this to reference both, "dashboard model" and "standard risk test model"